Projeto - Shark/index.html

```diff
@@ -29,7 +29,7 @@
                 </div>
                 <div class="hero__title">
                     <h1 class="shark_title">SHARKS</h1>
-                    <p>Unraveling the Mysteries of the Kings of the Oceans</p>
+                    <p>Unraveling the Mysteries of Diegao the Kings of the Oceans</p>

```

Comment:
Ao inves da palavra Diegao ser hard coded, poderia ser a partir de uma variavel.

```suggestion
const name = 'Diegao' ;

<p>Unraveling the Mysteries of ${name} the Kings of the Oceans</p>
```

https://angular.dev/guide/templates/reference-variables#example-4

Reply:
essa sugestao seria com intuito de tornar o codigo dinamico em breve, por exemplo.. quando fossemos resgatar o name atraves de uma API